samples/bluetooth/mesh/dfu/distributor/README.rst

```diff
@@ -167,7 +167,7 @@ A firmware image can be uploaded to the device in two ways:
 
 For out-of-band upload, the sample uses the image management subsystem of the :ref:`zephyr:mcu_mgr`.
 The management subsystem uses the Simple Management Protocol (SMP), provided by the Mcumgr library, to exchange commands and data between the SMP server (the sample device) and the SMP client.
-This sample supports Bluetooth Low Energy as the SMP transport.
+This sample supports Bluetooth Low Energy and UART as the SMP transport.
```

Comment:
Do you mean RTT here?

Reply:
No. This is for mcumgr: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/services/device_mgmt/mcumgr.html#configuring-the-transport. BLE transport allows to use nRF Connect Device Manager to interact with the device, or mcumgr client but with bluetooth controller running on host which requires a 3rd device, e.g. nRF52 with hci_uart sample (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/samples/bluetooth/hci_uart/README.html#using-the-controller-with-qemu-and-native-posix). UART transport allows to run mcumgr client and interact with the device over shell without 3rd-party component. Enabling RTT shell backend allows to execute shell commands on the sample using e.g. RTT Viewer. You can still execute shell commands over UART but using mcumgr.

Reply:
Ah sorry I mixed things up. You are right.